std/file.d

```diff
@@ -484,6 +464,22 @@ void append(R)(R name, const void[] buffer)
         writeImpl(null, name.tempCString!FSChar(), buffer, true);
 }
 
+///
+unittest
+{
+   scope(exit)
+   {
+       assert(exists("someUnqiueFilename"));
+       remove("someUnqiueFilename");
+   }
+
+   int[] a = [ 0, 1, 1, 2, 3, 5, 8 ];
+   write("someUnqiueFilename", a);
+   int[] b = [ 13, 21 ];
+   append("someUnqiueFilename", b);
+   assert(cast(int[]) read("someUnqiueFilename") == a ~ b);
```

Comment:
The filename is certainly unique, but it looks like a typo. :-)  `someUniqueFilename`, not `someUnqiueFilename`.